src/BugsnagUnity/Configuration.cs

```diff
@@ -54,15 +55,35 @@ public class Configuration : IMetadataEditor, IFeatureFlagStore
 
         internal OrderedDictionary FeatureFlags = new OrderedDictionary();
 
+
+        public List<Regex> RedactedKeys = new List<Regex>{new Regex(".*password.*",RegexOptions.IgnoreCase)};
         public bool KeyIsRedacted(string key)
         {
-            if (RedactedKeys == null || RedactedKeys.Length == 0)
+            if (RedactedKeys == null || RedactedKeys.Count == 0)
```

Comment:
Seems unneccesary to check length?

```suggestion
            if (RedactedKeys != null)
            {
                foreach (var regex in RedactedKeys)
                {
                    ....
                }
            }
```